package.json

```
@@ -89,6 +89,7 @@
     "rehype-slug": "^5.0.0",
     "rehype-stringify": "^9.0.0",
     "remark-cli": "^11.0.0",
+    "remark-directive": "^2.0.1",
```

```suggestion
    "remark-directive": "^2.0.0",
```

Let’s stick with the style of using clear ranges (except if we know stuff breaks)!